### DISCUSSION TITLE
How to configure what happens when users want to activate their paid access after reinstalling or using the extension among multiple browsers?

### DISCUSSION THREAD

**Discussion by luizen** [Glench/ExtPay#242] [Q&A] [answered]
In the extpay FAQ section, there is this question: 

>  **What happens if a paid user uninstalls and reinstalls an extension? Can paid users have an extension on multiple browsers?**
Developers can configure what happens for users wanting to activate their paid access after reinstalling or using the extension among multiple browsers. If reactivations are authorized, users can enter the email address they used to pay and will get a login link sent to their inbox. When they click the link in their email, the extension's paid access is reactivated.

However, I couldn't find anywhere how to configure this. **Anyone?** 

Basically I want to avoid receiving emails from paid users saying that they ran out of number of installations.

Thanks

**Comment by FiveMLuxe** [comment]
You can’t do anything. Ext pay is no longer developed or supported.

**Reply by Glench** [reply]
This is incorrect. Please don't spread misinformation. ExtPay works just fine and has been continually making extension developers money :)

**Reply by FiveMLuxe** [reply]
My apologies, just went from reading what others have said. Are you still actively developing this platform and plan to release enhancments?

**Reply by Glench** [reply]
Yep!

**Reply by FiveMLuxe** [reply]
Excellent. Looking forward to it. Is there any chance I can be a beta tester for the new version? We heavily rely on EXT Pay for our logging in / purchasing and would love to part of the testing. Thanks Glen!

**Comment by Glench** [comment, marked as answer]
Hey @luizen, for now if users run out of installs please just email me. In the future we hope to make this something the extension developer and users themselves can manage, but for now just email me.
